### PR TITLE
Updating copyright headers to match current best practices

### LINKED ISSUE #9: Add license header to top of all code files
See https://opensource.google/docs/releasing/licenses/

We can either use;
```
Copyright (C) 2020  The SymbiFlow Authors.

Use of this source code is governed by a ISC-style
license that can be found in the LICENSE file or at
https://opensource.org/licenses/ISC
```

Or put the full license at the top;
```
Copyright (C) 2020  The SymbiFlow Authors.

Permission to use, copy, modify, and/or distribute this software for any
purpose with or without fee is hereby granted, provided that the above
copyright notice and this permission notice appear in all copies.

THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
```

### PR DESCRIPTION
**Pull request by mkurc-ant**
Signed-off-by: Maciej Kurc <mkurc@antmicro.com>

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (C) 2019  SymbiFlow Project Authors. All rights reserved.
+Copyright (C) 2019  SymbiFlow Project Authors.
```
Comment (mithro) [inline LICENSE]: Expand copyright date.

File: setup.py [hunk 2]
```diff
@@ -1,3 +1,20 @@
+#
+# Copyright (C) 2019  The SymbiFlow Authors.
+#
```
Comment (mithro) [inline setup.py:5]: Please use the short form of the ISC license header

File: setup.py [hunk 3]
```diff
@@ -1,3 +1,20 @@
...
```
Comment (mithro) [inline setup.py:3]:
```
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
```

File: vtr_xml_utils/__main__.py [hunk 4]
```diff
@@ -1,4 +1,21 @@
...
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+# WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+# MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+# ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+# WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+# ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+# OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+#
+# SPDX-License-Identifier:  ISC
+#
```
Comment (mithro) [inline vtr_xml_utils/__main__.py]: Extra `#` at the end.

File: vtr_xml_utils/resources/convert-pb_type-attributes.xsl [hunk 5]
```diff
@@ -1,4 +1,23 @@
...
+ THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+ SPDX-License-Identifier:   ISC
+
```
Comment (mithro) [inline vtr_xml_utils/resources/convert-pb_type-attributes.xsl]: Extra newline here.

**Review by mithro — CHANGES_REQUESTED** [review]
Please go through the checklist at https://github.com/SymbiFlow/ideas/issues/47 and make sure everything is covered then re-request review.

**Comment by daniellimws** [comment]
The files inside https://github.com/SymbiFlow/vtr-xml-utils/tree/master/tests don't have the headers yet.

<blockquote><div> GitHub</div><div><strong><a href="https://github.com/SymbiFlow/vtr-xml-utils">SymbiFlow/vtr-xml-utils</a></strong></div><div>Contribute to SymbiFlow/vtr-xml-utils development by creating an account on GitHub.</div></blockquote>

File: LICENSE (hunk 1 above)
Comment (mkurc-ant) [reply LICENSE]: Fixed

File: setup.py (hunk 3 above)
Comment (mkurc-ant) [reply setup.py:3]: Fixed

File: vtr_xml_utils/__main__.py (hunk 4 above)
Comment (mkurc-ant) [reply vtr_xml_utils/__main__.py]: Removed

File: vtr_xml_utils/resources/convert-pb_type-attributes.xsl (hunk 5 above)
Comment (mkurc-ant) [reply vtr_xml_utils/resources/convert-pb_type-attributes.xsl]: Removed

File: setup.py (hunk 2 above)
Comment (mkurc-ant) [reply setup.py:5]: Changed everywhere

**Comment by mkurc-ant** [comment]
@daniellimws Added the license header to the test script
@mithro Should all xml files under the test directory also have a license? These are just data files for the tests.

**Comment by daniellimws** [comment]
There's also this in the checklist.

- [ ] Add or update the AUTHORS files to have the text as shown at https://opensource.google/docs/releasing/authors/

<blockquote><div>opensource.google</div><div><strong><a href="https://opensource.google/docs/releasing/authors/">AUTHORS File – opensource.google</a></strong></div></blockquote>

**Comment by mkurc-ant** [comment]
@daniellimws I've added the `AUTHORS` file.

File: AUTHORS [hunk 6]
```diff
@@ -0,0 +1,12 @@
+s is the list of vtr-xml-utils' significant contributors.
```
Comment (daniellimws) [inline AUTHORS]: `This`?

Comment (mkurc-ant) [reply AUTHORS]: Yep... Fixed.

File: AUTHORS [hunk 7]
```diff
@@ -0,0 +1,12 @@
+This is the list of vtr-xml-utils' significant contributors.
```
Comment (daniellimws) [inline AUTHORS]: `# This is ...`

**Review by daniellimws — CHANGES_REQUESTED** [review]

**Review by daniellimws — APPROVED** [review]

**Comment by daniellimws** [comment]
@mithro Looks like this can be merged already

**Comment by litghost** [comment]
Please rebase on master

**Comment by mkurc-ant** [comment]
@litghost @mithro Rebased.

**Review by mithro — APPROVED** [review]
LGTM!

**Pull request merged**